### DISCUSSION TITLE
Welcome to ComposeMultiplatformMediaPlayer Discussions!

### DISCUSSION THREAD

**Discussion by ctn-mobile-innovation** [Chaintech-Network/ComposeMultiplatformMediaPlayer#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PedroScar** [comment]
Hey everyone, amazing job on this project!
What you've accomplished here is truly impressive, and I’m confident it will have a significant impact on the KMP community.

I do have one suggestion. In the YouTube Compose feature, videos currently auto-start by default. Would it be possible to add an option to enable or disable auto-play?
